src/schedule/ucc_schedule_pipelined.c

```diff
@@ -14,6 +14,7 @@ static ucc_status_t ucc_frag_start_handler(ucc_coll_task_t *parent,
     ucc_schedule_t *frag               = ucc_derived_of(task, ucc_schedule_t);
     ucc_status_t    status;
 
+    task->start_time = parent->start_time;
```

Comment:
is it enough or we need to sert mask/flag for timeout on children task? (UCC_COLL_SET_TIMEOUT macro?)
Do you need to check if parent has timeout set?